optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-client/src/main/resources/org/optaplanner/workbench/screens/solver/client/resources/i18n/SolverEditorLookupConstants.properties

```diff
@@ -27,3 +27,8 @@ ALLOCATE_FROM_POOL=Allocate from pool
 NONE=None
 DECREASING_DIFFICULTY=Decreasing difficulty
 DECREASING_DIFFICULTY_IF_AVAILABLE=Decreasing difficulty if available
+
+HILL_CLIMBING=Hill climbing
+TABU_SEARCH=Tabu search
```

Comment:
In the docs, I 've taken the style of always capitalizing algorithm names, so "Tabu Search" instead of "Tabu search" and "First Fit Decreasing if available" instead of "First fit decreasing if available.
Do we want to follow that style here too?